src/components/MapComponent.js

```diff
@@ -1,4 +1,4 @@
-import React, { useEffect, useRef } from 'react';
+import { useEffect, useRef, useState } from 'react';
```

Comment:
seems that `useState` is never used, we could remove it here

Reply:
yes i will remove it.